googlemock/test/gmock_ex_test.cc

```diff
@@ -31,6 +31,7 @@
 
 // Tests Google Mock's functionality that depends on exceptions.
 
+#if GTEST_HAS_EXCEPTIONS
```

Comment:
Please move the #if after the #includes. After all the GTEST_HAS_EXCEPTIONS is defined in [the header](https://github.com/google/googletest/blob/41f0e24335d8874cc87420a6349bb1844e2a15de/googletest/include/gtest/internal/gtest-port.h#L475) included in the header file, otherwise it will never be defined

Thank you again